core/src/components/radio/radio.tsx

```diff
@@ -214,18 +214,21 @@ export class Radio implements ComponentInterface {
   private renderRadio() {
     const { checked, disabled, inputId, color, el, justify, labelPlacement, inheritedAttributes, hasLabel } = this;
     const mode = getIonMode(this);
+    const inItem = hostContext('ion-item', el);
 
     return (
       <Host
         onClick={this.onClick}
         class={createColorClasses(color, {
           [mode]: true,
-          'in-item': hostContext('ion-item', el),
-          interactive: true,
+          'in-item': inItem,
           'radio-checked': checked,
           'radio-disabled': disabled,
           [`radio-justify-${justify}`]: true,
           [`radio-label-placement-${labelPlacement}`]: true,
+          // Focus and active styling should not apply when the radio is in an item
+          'ion-activatable': !inItem,
+          'ion-focusable': !inItem,
```

Comment:
Why?

Reply:
This disabled the focus and active styling for the radio, when it is rendered inside of an item. Without it, the focused styles display, in addition to the item's focus styles:

![localhost_3333_src_components_radio_test_item(iPhone 12 Pro)](https://user-images.githubusercontent.com/13732623/211366425-f7fe284f-6c3d-4a98-aec8-c1164d90ab44.png)

This is inconsistent with how a radio behaves within an item on `main`:

![CleanShot 2023-01-09 at 12 13 22](https://user-images.githubusercontent.com/13732623/211367156-0c67ce5d-8b0d-47b4-9a79-b19510f1ff5f.png)

We do however, want focus styles when the radio is rendered by itself and then focused.